### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by shayfaber**

### CODE REVIEW

File: tests/base/account.spec.ts [hunk 1]
```diff
@@ -10,11 +10,26 @@ import globalSelector from './fixtures/during/selectors/global.json';
 import accountValue from './fixtures/during/input-values/account.json';
 import accountExpected from './fixtures/verify/expects/account.json';
 
+/**
```
Comment (dheesen) [inline tests/base/account.spec.ts]: As discussed, better to improve the naming of `test.describe` in rule 19.

File: tests/base/account.spec.ts [hunk 2]
```diff
@@ -39,6 +54,16 @@ test.describe('Test user flow', () => {
...
   }
 
+  /**
+   * @feature Magento 2 Login
+   *  @scenario User logs in on the Magento 2 web shop
+   *    @given I am on any Magento 2 page
+   *    @when I go to the login page
+   *    @and I fill in the required information
+   *    @and I click the 'Sign in' button
+   *    @then I should be logged in
```
Comment (dheesen) [inline tests/base/account.spec.ts]: We're currently not checking for a logged in status (would be a nice addition to check for cookies or something). Line 64 can be removed and line 65 will be the last line of this Gherkin block. Can you create an issue for checking the logged in cookie?

File: tests/base/account.spec.ts [hunk 3]
```diff
@@ -50,6 +75,16 @@ test.describe('Test user flow', () => {
...
   }
 
+  /**
+   * @feature Magento 2 Add an Address to Account
+   *  @scenario User adds an address to their account
+   *    @given I am logged in
+   *    @when I go to the page where I can update my address
+   *    @and I fill in the required information
+   *    @and I click the 'Save' button
+   *    @then My address should be updated
```
Comment (dheesen) [inline tests/base/account.spec.ts]: @shayfaber Address will be added here instead of updated right?

File: tests/base/account.spec.ts [hunk 4]
```diff
@@ -91,6 +136,16 @@ test.describe('Test user flow', () => {
     await accountPageTester.testPage();
   });
 
+  /**
```
Comment (dheesen) [inline tests/base/account.spec.ts:132]: Here should be added the scenario `@given I am subscribed now' etc. To also describe the scenario that we will test unsubscription.

File: tests/base/account.spec.ts [hunk 5]
```diff
@@ -187,6 +250,16 @@ test.describe('Test user flow', () => {
...
+   * @feature Magento 2 Update First Name and Last Name
+   *  @scenario User updates their name details
+   *    @given I am logged in
+   *    @when I navigate to the page where I can edit my first name and last name
+   *    @and I update the fields I wish to update (both fields)
+   *    @and I click the 'Save' button
+   *    @then My updated information should be saved
+   *    @and I should see a notification that my details have been updated.
```
Comment (dheesen) [inline tests/base/account.spec.ts:260]: @shayfaber Do you want to add here that we also test changing only first name?

File: tests/base/account.spec.ts [hunk 6]
```diff
@@ -206,6 +279,16 @@ test.describe('Test user flow', () => {
...
   });
 
+  /**
+   * @feature Magento 2 Delete Address
+   *  @scenario User removes their address
+   *    @given I am logged in
+   *    @when I navigate to the page where I can edit my address
+   *    @and I update the fields I wish to update (both fields)
+   *    @and I click the 'Save' button
+   *    @then My updated information should be saved
```
Comment (dheesen) [inline tests/base/account.spec.ts]: @shayfaber Will test here deletion.

**Review by dheesen — CHANGES_REQUESTED** [review]

File: tests/base/account.spec.ts (hunk 4 above)
Comment (shayfaber) [reply tests/base/account.spec.ts:132]: I'll add both scenario's under the feature.

**Review by dheesen — APPROVED** [review]

**Pull request merged**